examples/basic/add/run.py

```diff
@@ -0,0 +1,19 @@
+import numpy as np
+
+from oremda import Client as OremdaClient
+from oremda import Operator as OremdaOperator
+
+class AddOperator(OremdaOperator):
+    def kernel(self, input_data, parameters):
```

Comment:
@cjh1 Mentioned that we might provide a decorator that does a bunch of things like subclassing behind the scenes, so the user can just provide any plain old function. I think this is a really good idea also, so the user could do something like this:

```python
@operator
def add(input_data, parameters):
    return input_data + parameters.get('value', 0)
```

That reduces a lot of boilerplate.